From b31852c4caa36cc564e25adbdacfa534e1dfe7c0 Mon Sep 17 00:00:00 2001
From: =?utf8?q?Morten=20Johan=20S=C3=B8rvig?= <morten.sorvig@qt.io>
Date: Mon, 13 Jan 2020 14:48:32 +0100
Subject: [PATCH] Restore QHighDPIScaling::isActive() 5.13 behavior

In 5.13, isActive() would become true only if there
was a scale factor >1 present. This was accidentally
changed in 5.14, where isActive() becomes true whenever
e.g. AA_EneableHighDpiScaling is set, no matter the actual
scale factor values.

Change-Id: Iacbe2010cddbc3b9015ac24004ae2fe417d4f434
Fixes: QTBUG-80967
Reviewed-by: Friedemann Kleint <Friedemann.Kleint@qt.io>
---
 src/gui/kernel/qhighdpiscaling.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gui/kernel/qhighdpiscaling.cpp b/src/gui/kernel/qhighdpiscaling.cpp
index fde6bb0180b..671c2d93ef6 100644
--- a/src/gui/kernel/qhighdpiscaling.cpp
+++ b/src/gui/kernel/qhighdpiscaling.cpp
@@ -535,7 +535,7 @@ void QHighDpiScaling::updateHighDpiScaling()
             ++i;
         }
     }
-    m_active = m_globalScalingActive || m_screenFactorSet || m_usePixelDensity;
+    m_active = m_globalScalingActive || m_screenFactorSet || m_pixelDensityScalingActive;
 }
 
 /*
-- 
2.16.3

